packages/RichText-Core.package/RTETextComponent.class/instance/importTextFontIfNecessary.st

```diff
@@ -0,0 +1,7 @@
+initialization
+importTextFontIfNecessary
```

Comment:
Could you write a test for that? Would be cool for our coverage. If it unpossible it's okay.

Reply:
actually don't need that function anymore, i'll fix it in a quick commit

Reply:
isn't on my version either